### PR TITLE
Minor Fixes and Updates

### PR DESCRIPTION
**Pull request by Jacques00**
* Some minor corrections.
* Insert in time stamps for most (if not all) eventBuffer messages. Color coded for awesome.
* Remember to parse the messages before adding them in the buffer, for accuracy.
* Hopefully fix Riley issue with a disableExploreEvents() function.

### CODE REVIEW

File: classes/GameData/CombatContainer.as [hunk 1]
```diff
@@ -274,7 +274,7 @@ package classes.GameData
 					h.triggerAlarm(true);
 				}
 			}
-			if(pc.shields() <= 0 && !pc.accessory.hasFlag(GLOBAL.ITEM_FLAG_INTERNAL_POWER))
+			if(pc.shields() <= 0 && pc.accessory.hasFlag(GLOBAL.ITEM_FLAG_COMBAT_DRONE) && !pc.accessory.hasFlag(GLOBAL.ITEM_FLAG_INTERNAL_POWER))
```
Comment (Fenoxo) [inline classes/GameData/CombatContainer.as]: Pretty sure this will actually prevent these messages from working with "default" non-accessory drones.

Comment (Jacques00) [inline classes/GameData/CombatContainer.as]: True, I also thought about that too--might need to elaborate the conditional a little more.

Comment (Fenoxo) [inline classes/GameData/CombatContainer.as]: The original conditional worked perfectly so far as I can see. The "INTERNAL_POWER" flag only exists for drones and should only be used for drones.

Comment (Fenoxo) [inline classes/GameData/CombatContainer.as]: I take that back. I'll have to work on this later >.>

Comment (Jacques00) [inline classes/GameData/CombatContainer.as]: I might have a fix, do I don't know if perk drones and accessory drones overlap or one replaces the other. But fix is in the new pull request!

**Pull request merged**